src/Service/AuthorizationService.php

```diff
@@ -51,4 +56,24 @@ public function isAllowed(string $right): bool
         }
         return true;
     }
+
+    /**
+     * Checks if the user name is 'developer' and the developer module is active
+     *
+     * The check if the module is active is done using permissions. The module
+     * sets the permission 'all' for the group 'developers'. So if this permission
+     * is not set, the module is not active.
+     *
+     * @return bool
+     */
+    private function isDeveloperModuleRequest()
```

Comment:
In my opinion this method should also go to the developer module itself. To make it callable, we could fire a grapqhlIsAllowed Event (or similar). The developer module can register to that (everybody else could also) and you could allow the right from there. I would like to discuss that with you @mtkoltan

Reply:
as discussed I will implement the suggestions in this module